src/view-compiler.js

```diff
@@ -158,7 +158,7 @@ export class ViewCompiler {
       return this._compileElement(node, resources, instructions, parentNode, parentInjectorId, targetLightDOM);
     case 3: //text node
       //use wholeText to retrieve the textContent of all adjacent text nodes.
-      let expression = resources.getBindingLanguage(this.bindingLanguage).inspectTextContent(resources, node.wholeText);
+      let expression = resources.getBindingLanguage(this.bindingLanguage).inspectTextContent(resources, node.wholeText || node.data);
```

Comment:
@jdanyow How much work is involved in getting the PR to jsdom? We could do this in the mean time, but I'm a bit worried about how it might affect performance. Could we polyfill the property instead? Literally, monkey patch it.